lib/directories.zsh

```diff
@@ -24,10 +24,11 @@ alias rd=rmdir
 alias d='dirs -v | head -10'
 
 # List directory contents
-alias lsa='ls -lah'
-alias l='ls -lah'
+# alias lsa='ls -lah'
+alias l='ls'
 alias ll='ls -lh'
-alias la='ls -lAh'
+alias la='ls -Ah'
+alias lla='ls -lah'
```

Comment:
Please see previous comment.
`lla` and `lsa`, which is clear to know it's result is for `ls -l -a` ?
Meanwhile, `lsa` looks like `ls -a` but not `ls -l -a`. Isn't it?